owner-onboarding-server/src/main.rs

```diff
@@ -335,6 +401,41 @@ async fn main() -> Result<()> {
             .context("Error converting owner public key to PK")?
     };
 
+    // Voucher registration times
+    let ov_registration_period = match settings.ov_registration_period {
+        Some(value) => {
+            if value == 0 {
+                bail!("ov_registration_period cannot be 0");
+            }
+            value
+        }
+        None => {
+            log::info!(
+                "Setting a default ov_registration_period of {DEFAULT_REGISTRATION_PERIOD} seconds"
+            );
+            DEFAULT_REGISTRATION_PERIOD
+        }
+    };
+    let ov_re_registration_window = match settings.ov_re_registration_window {
+        Some(value) => {
+            if value == 0 {
+                bail!("ov_re_registration_window cannot be 0");
+            }
+            value
+        }
+        None => {
+            log::info!("Setting a default ov_re_registration_window of {DEFAULT_RE_REGISTRATION_WINDOW} seconds");
+            DEFAULT_RE_REGISTRATION_WINDOW
+        }
+    };
+
+    if ov_re_registration_window >= ov_registration_period {
+        bail!(
+            "ov_re_registration_window ({ov_re_registration_window}) must be smaller than ov_registration_period ({ov_registration_period})");
+    } else {
+        log::info!("Server configured with an OV registration period of {ov_registration_period} seconds, OV re-registration window set to {ov_re_registration_window} seconds")
+    }
+
```

Comment:
@mmartinv I'm guessing that this is it, let me explain since I did not comment.
I have a pending issue to add sqlite support as an optional argument when running aio (#642 ), as part of that I would have to change the passed arguments to the aio command, and it makes sense to add a way to configure other things (such as this new registration window there), so, yeah I made a mental note of this but did not comment. My apologies.